logging.yaml

```diff
@@ -1,19 +1,12 @@
 ---
 version: 1
 
-# Leave the new structured log handler turned off until we figure out log storage destination.
-# Currently written as a file handler for structured logging proof of concept.
-
-# root:
-#   handlers:
-#     - fileHandler
```

Comment:
Do we still need this part of the config?

Reply:
Yup totally, my bad - dropped this on accident while rebasing after the reversion.